contracts/consensus/Consensus.sol

```diff
@@ -964,6 +1101,17 @@ contract Consensus is MasterCopyNonUpgradable, CoreLifetimeEnum, MosaicVersion,
             _dislocation,
             _metablockHash
         );
+
+        CommitteeI committee = committees[_metablockHash];
+
+        emit MetablockCommitteeFormed(
+            address(committee),
+            _metablockHash,
+            _metachainId,
+            _committeeFormationBlockHeight,
+            committeeSize,
+            committee.quorum()
```

Comment:
`committee.quorum()` can be evaluated from `committeeSize`

Reply:
Updated